packages/dns-discovery/src/enrtree.ts

```diff
@@ -39,21 +39,22 @@ export class ENRTree {
     // of the record content, excluding the `sig=` part, encoded as URL-safe base64 string
     // (Trailing recovery bit must be trimmed to pass `ecdsaVerify` method)
     const signedComponent = root.split(" sig")[0];
+
     const signedComponentBuffer = utf8ToBytes(signedComponent);
     const signatureBuffer = fromString(rootValues.signature, "base64url").slice(
       0,
       64
     );
 
+    return rootValues.eRoot;
+
     const isVerified = verifySignature(
       signatureBuffer,
       keccak256(signedComponentBuffer),
       new Uint8Array(decodedPublicKey)
     );
 
     if (!isVerified) throw new Error("Unable to verify ENRTree root signature");
-
-    return rootValues.eRoot;
   }
```

Comment:
This is done because the signature, for unknown reasons ([tracked here](https://github.com/waku-org/js-waku/issues/1085))

Reply:
This has been resolved by updating the `pubkey`